### PR TITLE
Trivial get endpoint

### PR DESCRIPTION
**Pull request by michalc**

### CODE REVIEW

File: core/app.py [hunk 1]
```diff
@@ -80,13 +82,44 @@ def parse_feed_config(feed_config):
...
+    async def handle(_):
+        return web.json_response({'secret': 'to-be-hidden'})
+
+    app_logger.debug('Creating listening web application...')
+    app = web.Application(middlewares=[
+        authenticator(ip_whitelist, incoming_key_pairs),
+        authorizer(),
+    ])
+    app.add_routes([
+        web.post('/v1/', handle),
```
Comment (lgarvey) [inline core/app.py]: Does your API need url based versioning?

Comment (michalc) [reply core/app.py]: There is essentially a requirement to expect changes to the data/formats involved. It doesn't have to be URL-based versioning (so alternatives welcome!), but I think something to allow for backwards incompatible changes to be released while keeping the old service running is good.

There is also probably a good YAGNI argument though: so happy to remove for now, as nothing forbids adding it in later?

Comment (michalc) [reply core/app.py]: Ah since you've approved, will merge, but happy to chat offline!

Comment (michalc) [reply core/app.py]: ^Being silly. You haven't approved!

Comment (lgarvey) [reply core/app.py]: I did approve it. Let's have a chat about versioning offline.

**Pull request merged**